src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -376,7 +375,7 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
     return (
       <div className="container-fluid container-cards-pf">
-        <Grid gutter={GutterSize.md}>
+        <Grid gutter={'md'}>
           <GridItem span={editorSpan}>
```

Comment:
This error is a regresion from PF 

Opened Issue https://github.com/patternfly/patternfly-react/issues/4037

We can go ahead